symplyphysics/laws/thermodynamics/stefan_boltzmann_law.py

```diff
@@ -0,0 +1,31 @@
+from sympy import (Eq, solve)
```

Comment:
We do not use marketing names for the laws, like `Newton's Second Law`. We use descriptive law names: what physical value can be obtained from what. So the name of the file should be smth like `irradiation_of_black_body_from_temperature.py`